server/resp/src/main/java/org/infinispan/server/resp/filter/RespTypeFilterConverter.java

```diff
@@ -26,7 +27,7 @@ public RespTypeFilterConverter(byte ordinal) {
       this.type = RespTypes.fromOrdinal(ordinal);
    }
 
-   @ProtoField(number = 1, defaultValue = "0")
+   @ProtoField(1)
    byte ordinal() {
```

Comment:
I'm wondering why not sending the add the Proto annotation to the enum and sending it?